halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentEnvironment.java

```diff
@@ -94,6 +94,7 @@ public static Size fromString(String name) {
   private String accountName;
   private Consul consul = new Consul();
   private Vault vault = new Vault();
+  private String location = "spinnaker";
```

Comment:
I made a comment of this in the other PR, but every provider needs to implement some interface that returns a default deployment location that is referenced/read when you configure which account you're deploying to. The reason being, "spinnaker" only makes sense for k8s - when deploying to GCE you'd need something like "us-central1-f".